### PR TITLE
Improvements to the memcache compose file

### PR DESCRIPTION
**Pull request by damienmckenna**
## The Problem/Issue/Bug:
The docker-compose.memcache.yaml file's comments needs a little tidying.

## How this PR Solves The Problem:
This tidies up the file to be easier to follow.

## Related Issue Link(s):
#1549

## Release/Deployment notes:
n/a

### CODE REVIEW

File: pkg/servicetest/testdata/services/docker-compose.memcached.yaml [hunk 1]
```diff
@@ -1,20 +1,30 @@
-# ddev memcached recipe file
+# DDev memcached recipe file.
```
Comment (rfay) [inline pkg/servicetest/testdata/services/docker-compose.memcached.yaml]: I guess if we're going to be official it should be `DDEV-Local`

File: pkg/servicetest/testdata/services/docker-compose.memcached.yaml [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
 #
-# To use this in your own project: Copy this file to your project's .ddev folder.
-# Edit the 'command' settings to change CLI flags sent to memcached.
-# Defaults to '-m 128' to allocate 128 MB of memcached storage.
+# To use this in your own project:
+# 1. Copy this file to your project's ".ddev" directory.
+# 2. Launch "ddev start".
+# 3. Configure the project to look for memcached at the hostname "memcache" and
```
Comment (rfay) [inline pkg/servicetest/testdata/services/docker-compose.memcached.yaml]: I think the hostname is "memcached" right? At least we should make it that way.

File: pkg/servicetest/testdata/services/docker-compose.memcached.yaml [hunk 3]
```diff
@@ -1,20 +1,30 @@
-# ddev memcached recipe file
+# DDev memcached recipe file.
 #
-# To use this in your own project: Copy this file to your project's .ddev folder.
-# Edit the 'command' settings to change CLI flags sent to memcached.
-# Defaults to '-m 128' to allocate 128 MB of memcached storage.
```
Comment (jeffsheltren) [inline pkg/servicetest/testdata/services/docker-compose.memcached.yaml]: I would like to keep this comment somewhere so it's obvious what the default memory allocation is. Could be down below with the actual arguments if that makes more sense.

**Comment by rfay** [comment]
There are a couple of minor open comments on this; I'll be happy to pull it when you say it's ready @damienmckenna ; if you want me to address those I can also do it.

**Comment by rfay** [comment]
I took the liberty of pushing the minor review comment changes in. Thanks so much for this!

**Pull request merged**